modules/nvidia_plugin/src/error.cpp

```diff
@@ -20,7 +25,7 @@ template <typename T>
 
 [[gnu::cold, noreturn]] void throw_ov_exception(const std::string& msg,
                                                 const std::experimental::source_location& location) {
-    throw_exception<ov::Exception>(msg, location);
+    throw_exception<OVExceptionWrapper>(msg, location);
```

Comment:
we could create specialization of this `throw_exception` which throws via `Exception::create`